### PR TITLE
fix(copy): change 'unconfigure' to 'delete election data'.

### LINKED ISSUE #1577: VxScan: "unconfigure" text should match VxCentralScan and say "delete all data"
VxScan "unconfigure" is a bit like the VxMark, but the VxMark doesn't have critical data, whereas VxScan does, so the button should be more like the VxCentralScan one, "delete all data" with all the warnings.

### PR DESCRIPTION
**Pull request by benadida**
fixes #1577

### CODE REVIEW

File: frontends/precinct-scanner/src/screens/election_manager_screen.tsx [hunk 1]
```diff
@@ -229,7 +229,7 @@ export function ElectionManagerScreen({
         <Modal
           content={
             <Prose>
-              <h1>Unconfigure Machine?</h1>
+              <h1>Delete Election Data?</h1>
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.tsx:232]:
```suggestion
              <h1>Delete All Election Data?</h1>
```

File: frontends/precinct-scanner/src/app.test.tsx [hunk 2]
```diff
@@ -457,7 +457,7 @@ test('election manager and poll worker configuration', async () => {
   await advanceTimersAndPromises(1);
   card.insertCard(electionManagerCard, electionSampleDefinition.electionData);
   await authenticateElectionManagerCard();
-  fireEvent.click(await screen.findByText('Unconfigure Machine'));
+  fireEvent.click(await screen.findByText('Delete Election Data from VxScan'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/app.test.tsx:460]:
```suggestion
  fireEvent.click(await screen.findByText('Delete All Election Data from VxScan'));
```

File: frontends/precinct-scanner/src/app.test.tsx [hunk 3]
```diff
@@ -467,8 +467,8 @@ test('election manager and poll worker configuration', async () => {
       'Do you want to remove all election information and data from this machine?'
     )
   ).toBeNull();
-  fireEvent.click(await screen.findByText('Unconfigure Machine'));
-  fireEvent.click(await screen.findByText('Unconfigure'));
+  fireEvent.click(await screen.findByText('Delete Election Data from VxScan'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/app.test.tsx:470]:
```suggestion
  fireEvent.click(await screen.findByText('Delete All Election Data from VxScan'));
```

File: frontends/precinct-scanner/src/app_unhappy_paths.test.tsx [hunk 4]
```diff
@@ -103,8 +103,8 @@ test('services/scan fails to unconfigure', async () => {
   fireEvent.click(screen.getByText('6'));
   await screen.findByText('Election Manager Settings');
 
-  fireEvent.click(await screen.findByText('Unconfigure Machine'));
-  fireEvent.click(await screen.findByText('Unconfigure'));
+  fireEvent.click(await screen.findByText('Delete Election Data from VxScan'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/app_unhappy_paths.test.tsx:106]:
```suggestion
  fireEvent.click(await screen.findByText('Delete All Election Data from VxScan'));
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx [hunk 5]
```diff
@@ -166,8 +166,8 @@ test('unconfigure ejects a usb drive when it is mounted', () => {
     </AppContext.Provider>
   );
 
-  fireEvent.click(screen.getByText('Unconfigure Machine'));
-  fireEvent.click(screen.getByText('Unconfigure'));
+  fireEvent.click(screen.getByText('Delete Election Data from VxScan'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx:169]:
```suggestion
  fireEvent.click(screen.getByText('Delete All Election Data from VxScan'));
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx [hunk 6]
```diff
@@ -194,8 +194,8 @@ test('unconfigure does not eject a usb drive that is not mounted', async () => {
...
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx:197]:
```suggestion
  fireEvent.click(screen.getByText('Delete All Election Data from VxScan'));
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx [hunk 7]
```diff
@@ -222,7 +222,7 @@ test('unconfigure button is disabled when the machine cannot be unconfigured', (
     </AppContext.Provider>
   );
 
-  fireEvent.click(screen.getByText('Unconfigure Machine'));
+  fireEvent.click(screen.getByText('Delete Election Data from VxScan'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx:225]:
```suggestion
  fireEvent.click(screen.getByText('Delete All Election Data from VxScan'));
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.tsx [hunk 8]
```diff
@@ -186,13 +186,13 @@ export function ElectionManagerScreen({
             <span role="img" aria-label="Warning">
               ⚠️
             </span>{' '}
-            Unconfigure Machine
+            Delete Election Data from VxScan
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.tsx:189]:
```suggestion
            Delete All Election Data from VxScan
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.tsx [hunk 9]
```diff
@@ -186,13 +186,13 @@ export function ElectionManagerScreen({
...
           </Button>
         </p>
         {!scannerStatus.canUnconfigure && (
           <p>
-            You must &quot;Export Backup&quot; before you may unconfigure the
-            machine.
+            You must &quot;Export Backup&quot; before you can delete delection
+            data from VxScan.
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.tsx:195]: typo
```suggestion
            You must “Export Backup” before you can delete election
            data from VxScan.
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.tsx [hunk 10]
```diff
@@ -239,7 +239,7 @@ export function ElectionManagerScreen({
           actions={
             <React.Fragment>
               <Button danger onPress={handleUnconfigure}>
-                Unconfigure
+                Yes, Delete
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.tsx:242]:
```suggestion
                Yes, Delete All
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx [hunk 11]
```diff
@@ -194,8 +194,8 @@ test('unconfigure does not eject a usb drive that is not mounted', async () => {
     </AppContext.Provider>
   );
 
-  fireEvent.click(screen.getByText('Unconfigure Machine'));
-  fireEvent.click(screen.getByText('Unconfigure'));
+  fireEvent.click(screen.getByText('Delete Election Data from VxScan'));
+  fireEvent.click(screen.getByText('Yes, Delete'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx:198]:
```suggestion
  fireEvent.click(screen.getByText('Yes, Delete All'));
```

File: frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx [hunk 12]
```diff
@@ -166,8 +166,8 @@ test('unconfigure ejects a usb drive when it is mounted', () => {
...
```
Comment (beausmith) [inline frontends/precinct-scanner/src/screens/election_manager_screen.test.tsx:170]:
```suggestion
  fireEvent.click(screen.getByText('Yes, Delete All'));
```

File: frontends/precinct-scanner/src/app_unhappy_paths.test.tsx [hunk 13]
```diff
@@ -103,8 +103,8 @@ test('services/scan fails to unconfigure', async () => {
...
+  fireEvent.click(await screen.findByText('Yes, Delete'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/app_unhappy_paths.test.tsx:107]:
```suggestion
  fireEvent.click(await screen.findByText('Yes, Delete All'));
```

File: frontends/precinct-scanner/src/app.test.tsx [hunk 14]
```diff
@@ -467,8 +467,8 @@ test('election manager and poll worker configuration', async () => {
...
+  fireEvent.click(await screen.findByText('Yes, Delete'));
```
Comment (beausmith) [inline frontends/precinct-scanner/src/app.test.tsx:471]:
```suggestion
  fireEvent.click(await screen.findByText('Yes, Delete All'));
```

**Review by beausmith — APPROVED** [review]
Looks good, but for clarity and differentiation from the other red buttons next to it, we should add the word "All". 

Also, there is one typo.

**Comment by beausmith** [comment]
Also https://github.com/votingworks/vxsuite/issues/1577 says to use "delete all data" 🤷‍♂️

File: frontends/precinct-scanner/src/screens/election_manager_screen.tsx (hunk 9 above)
Comment (benadida) [reply frontends/precinct-scanner/src/screens/election_manager_screen.tsx:195]: you don't like delete delection?

**Pull request merged**